build-system/tasks/compile.js

```diff
@@ -310,18 +310,28 @@ function compile(entryModuleFilenames, outputDir,
 };
 
 function patchRegisterElement() {
+  var file;
   // Copies document-register-element into a new file that has an export.
   // This works around a bug in closure compiler, where without the
   // export this module does not generate a goog.provide which fails
   // compilation.
   // Details https://github.com/google/closure-compiler/issues/1831
   const patchedName = 'build/patched-module/document-register-element' +
-      '/build/document-register-element.max.js';
+      '/build/document-register-element.node.js';
   if (!fs.existsSync(patchedName)) {
-      fs.writeFileSync(patchedName,
-          fs.readFileSync(
-              'node_modules/document-register-element/build/' +
-              'document-register-element.max.js') +
-          '\n\nexport function deadCode() {}\n');
-    }
+    file = fs.readFileSync(
+        'node_modules/document-register-element/build/' +
+        'document-register-element.node.js').toString();
+    // CommonJS support for closure does not wrap the module
+    // and inject `global` like browserify and other node module loaders
+    // so we nee to change this to `self` or `window`.
+    file = file.replace('installCustomElements(global);',
```

Comment:
Why not just only have exported function and call it directly from `polyfill.js`?